src/components/Books.js

```diff
@@ -1,27 +1,31 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/booksSlice';
+import Progress from './Progress';
 
-const Book = ({
-  title, author, itemId, category,
-}) => {
+const Book = (
+  {
+    title, author, itemId, category,
+  },
+) => {
   const dispatch = useDispatch();
 
   return (
-    <div>
-      <div>
-        <p>{category}</p>
-        <h2>{title}</h2>
-        <p>{author}</p>
-        <div>
-          <button
-            type="button"
-            onClick={() => dispatch(removeBook(itemId))}
-          >
-            Remove
-          </button>
+    <div className="book-container">
+      <div className="book-info">
+        <p className="book-category">{category}</p>
+        <h2 className="book-title">{title}</h2>
+        <p className="book-author">{author}</p>
+        <div className="controls-container">
+          <button className="book-control" type="button">Comments</button>
+          <button className="book-control" type="button" onClick={() => dispatch(removeBook(itemId))}>Remove</button>
+          <button className="book-control" type="button">Edit</button>
         </div>
       </div>
+      <div className="book-extra">
+        <Progress progress="75%" />
+        <div className="divider" />
```

Comment:
As you can see in the screenshot, The part on the right is missing, Kindly add that 👍 

### [YOUR DESIGN]
<img width="411" alt="Screenshot 2023-06-17 at 11 46 59 PM" src="https://github.com/MarcoDDM/booksStore/assets/83048208/c97f60eb-b970-495a-acb7-e17103e17ed4">

### [REQUIRED DESIGN]
<img width="408" alt="Screenshot 2023-06-17 at 11 48 46 PM" src="https://github.com/MarcoDDM/booksStore/assets/83048208/ff50bb5c-012e-41aa-8b26-955618b8864a">

Reply:
The sidebar with the css was over that section and I did not see it. Thanks for your suggestion.